Bottom: 2135394d07ba2919cdce6c8340af54ffdc5a5ee6
Top:    047dd99889de634ca308a8f7c5e9053dbc0a343b
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-15 16:46:57 +0800

Refresh of virito-net-keep-old

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 3eb732f..9ff959c 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1846,7 +1846,7 @@ static int virtnet_xdp_set(struct net_device *dev, struct bpf_prog *prog)
 
 virtio_queue_err:
 	/* Should not happen, after reset, all queue pairs were
-	 * enabled by defualt.
+	 * enabled by default.
 	 */
 	vi->curr_queue_pairs = vi->max_queue_pairs;
 	netif_set_real_num_tx_queues(dev, vi->curr_queue_pairs);
